source/includes/stackpath/_custom_rules.md

```diff
@@ -141,17 +141,110 @@ Attributes | &nbsp;
+`conditions`<br/>*Array[Object]* | The conditions required for the WAF engine to trigger the rule. All conditions must pass for the rule to trigger. Only for rule of type `WAF`.
 `conditions.type`<br/>*string* | Type of condition, one of: `IP`, `IP_RANGE`, `URL`, `USER_AGENT`, `HEADER`, `HTTP_METHOD`, `FILE_EXTENSION`, `CONTENT_TYPE`, `COUNTRY` or `ORGANIZATION`.
-`conditions.operation`<br/>*string* | Operation of the condition, one of: `EQUAL`, `NOT_EQUAL`, `CONTAINS`, `NOT_CONTAINS`, `BETWEEN` or `NOT_BETWEEN`.
-`conditions.header`<br/>*string* | Value of the header. Only for condition operation of type `HEADER`.
+`conditions.operation`<br/>*string* | Operation of the condition, one of: <ul><li>`EQUAL`, `NOT_EQUAL` for all condition type except `IP_RANGE`</li>, <li>`CONTAINS`, `NOT_CONTAINS` for condition type `HEADER`, `URL` and `USER_AGENT`</li><li>`BETWEEN` or `NOT_BETWEEN` only for condition type `IP_RANGE`</li></ul>.
+`conditions.header`<br/>*string* | Value of the header. Only for condition of type `HEADER`.
 `conditions.value`<br/>*string* | Value for which the condition holds.
-`conditions.endValue`<br/>*string* | Second value of the condition. Only for condition operation of type `IP_RANGE`.
\ No newline at end of file
+`conditions.endValue`<br/>*string* | Second value of the condition. Only for condition of type `IP_RANGE`.
+
+
+
+
+<!-------------------- CREATE A CUSTOM RULE -------------------->
+
+#### Create a custom rule
+
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -d "request_body" \
+    "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/customrules?siteId=0a57855b-26d8-4e8f-8b77-429997c7c5fb"
+```
+> Request body example for a custom rule with conditions:
+
+```json
+{
+  "name": "firewall-allow",
+  "notes": "some firewall note",
+  "type": "WAF",
+  "action": "BLOCK",
+  "actionDuration": "0s",
+  "statusCode": "FORBIDDEN_403",
+  "conditions": [
+    {
+      "type": "IP",
+      "operation": "EQUAL",
+      "value": "120.1.1.1"
+    }
+  ]
+}
+```
+
+> Request body example for a custom role with request rate:
+
+```json
+{
+  "name": "firewall-deny",
+  "notes": "Deny rule",
+  "type": "REQUEST_RATE",
+  "enabled": true,
+  "action": "BLOCK",
+  "actionDuration": "0s",
+  "statusCode": "TOO_MANY_REQUESTS_429",
+  "nbRequest": 20,
+  "duration": 60,
+  "pathRegExp": "/",
+  "httpMethods": ["GET"],
+  "ipAddresses": ["192.168.2.1"]
+}
+```
+> The above commands return a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/customrules?siteId=<a href="#stackpath-site">:siteId</a></code>
+
+Create a site in a given [environment](#administration-environments).
```

Comment:
Create a custom rule for a site in a given

Reply:
f4bd92d0ac1fe1446e61d38f51bddcc8a79a2f11

Reply:
e616ec87e506a501b0c2971428f2e45758bd052f